waltz-common/src/main/java/com/wepay/waltz/common/message/MessageCodecV0.java

```diff
@@ -120,7 +120,7 @@ public void encode(Message msg, MessageAttributeWriter writer) {
 
             case MessageType.MOUNT_RESPONSE:
                 MountResponse mountResponse = (MountResponse) msg;
-                writer.writeBoolean(mountResponse.partitionReady);
+                writer.writeInt(mountResponse.partitionState);
```

Comment:
This breaks compatibility. Do like this instead.
```
writer.writeBoolean(mountResponse.partitionState == PartitionState.READY);
```

Reply:
Perhaps if partitionState = CLIENT_AHEAD, we shouldn't send anything. Currently the message will be decoded as NOT_READY leading to more and more mounting calls from the client